rllib/core/optim/tests/test_rl_optimizer.py

```diff
@@ -2,34 +2,21 @@
 import pytest
```

Comment:
The goal is to minimize the total number of external components in these core unittests so that they won't fail because of the introduced bugs in other pieces. (truly be a unittest for what they are supposed to test)